specs/search/helpers/replaceAllObjects.yml

```
@@ -19,19 +19,24 @@ method:
     operationId: replaceAllObjects
     summary: Replace all records in an index
     description: |
-      Replace all records from your index with a new set of records.
-
-      This method lets you replace all records in your index without downtime. It performs these operations:
-        1. Copy settings, synonyms, and rules from your original index to a temporary index.
-        2. Add your new records to the temporary index.
-        3. Replace your original index with the temporary index.
-
-      Use the safe parameter to ensure that these (asynchronous) operations are performed in sequence.
-      If there's an error duing one of these steps, the temporary index won't be deleted.
-      This operation is rate-limited.
+      This method replaces all records in an index without interrupting ongoing searches.
+      
+      It combines [batch](https://www.algolia.com/doc/rest-api/search/batch) and [copy/move](https://www.algolia.com/doc/rest-api/search/operation-index) index operations:
+      1. Copy settings, synonyms, and rules to a temporary index.
+      2. Add the records from the `objects` parameter to the temporary index.
+      3. Replace the original index with the temporary one.
+      
+      If there's an error during one of these steps, the temporary index won't be deleted.
+      You can add the `deleteIndex` ACL to your API key to allow automatic deletion of the temporary index in case of failure.
```

```suggestion
      If there's an error during one of these steps,
      the temporary index is deleted if your API key has the `deleteIndex` ACL.
```

I think this makes it clear.